authors/apps/articles/utils/model_helpers.py

```diff
@@ -9,3 +9,16 @@ def get_single_article_using_slug(slug):
         return obj
     except Article.DoesNotExist:
         return None
+
+
+def get_all_available_tags():
```

Comment:
Do you think this is the best way you could have implemented this method?
Is it, for example, possible to do some kind of `aggregate` on the `tag_list` field?

Reply:
Thanks for the feedback @Karuhanga and yes I think that's the best way to implement this method because the essence of this method was not to aggregate available tags but to all available tags for articles in the DB. This, for example, is good for suggestions that could be made for our API consumers to their clients as they can filter all available tags with a key-up on a client input in the front-end to suggest available tag names for their article.
I hope this makes more clarity on my method.
Thanks again for the feedback